playground/AWS/AWS.AppHost/Properties/launchSettings.json

```diff
@@ -9,7 +9,8 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
         "DOTNET_ENVIRONMENT": "Development",
-        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "http://localhost:16216"
+        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "http://localhost:16216",
+        "ASPIRE_ALLOW_UNSECURED_TRANSPORT": "true"
```

Comment:
If we are putting this in all our playground apps, do we also need it in the templates?

Reply:
Or, if this isn't supposed to be in the templates, but instead we are supposed to use HTTPS, shouldn't all the playground apps be updated to use HTTPS instead?

Reply:
Not going in templates and the goal should be to update all the playground apps (or maybe all but one) to use HTTPS.

Reply:
I'll update the PR once I am back online this morning.

Reply:
I agree, one should use https, all of the rest should be http with this flag.

Reply:
Not sure who you're agreeing with, but I said the exact opposite :smile:

Reply:
#confused

Reply:
LOL

Reply:
OK I've got a http and https launch profile in all our launch settings file with settings that mimic (for the most part) what we get from the templates. There are some exceptions in some playgrounds for specific reasons.